src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -20495,15 +20512,15 @@ public void PartialMethodsConsiderRefKindDifferences_NoneWithOut()
         {
             CreateCompilation(@"
 partial class C {
-    partial void M(int i);
-    partial void M(out int i) { i = 0; }  
```

Comment:
Just curious - is there a reason to change the test (from no accessibility to `internal`)? If not, please consider reverting even though behavior has changed.

Reply:
Reverted, thanks. This was more of an artifact from the way this change originally worked.